lib/zstd_stats.h

```diff
@@ -0,0 +1,164 @@
+/*
+    zstd - standard compression library
```

Comment:
Clear enough. No pb on compilation.
I'm just wondering about source tree.

Currently, legacy support is within its own directory, because we don't want to include that in the general case.
I'm wondering if we should do the same for debug-only files.